core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2125,6 +2130,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
               case Some(ba) if !billingProject.invalidBillingAccount => ba
               case _ => throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, s"Billing Account is missing or invalid for Billing Project: ${billingProject}"))
             }
+            val workspaceName = WorkspaceName(workspaceRequest.namespace, workspaceRequest.name)
```

Comment:
maybe outside of the scope of this ticket, but the case class `WorkspaceName` seems ambiguous, especially when one of the params is the workspace name. I wonder if we can rename this FullWorkspaceName or at least rename the val :/

Reply:
@andy7i, yeah, certainly outside the scope of this ticket.  I'm sure this is used in a ton of places but I 100% agree with you that the name is confusing and it throws me off all the time and I'd LOVE to change it.  Can you make a tech debt ticket?

Reply:
yeah definitely -- created https://broadworkbench.atlassian.net/browse/CA-1264